src/app/static/src/js/components/workflowWizard/step.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+    <div v-if="active && hasValidComponent">
+        <div>
+            <slot></slot>
+        </div>
+        <div class="pt-4">
+            <button id="cancel-workflow" type="button" class="btn btn-sm btn-secondary"
+                    @click="cancel">Cancel
+            </button>
+            <button id="previous-workflow" v-if="buttonVisibility.back" type="button"
+                    class="btn btn-sm btn-primary"
+                    @click="back">Back
+            </button>
+            <button id="next-workflow" type="button" class="btn btn-sm btn-success"
+                    :class="{disabled: !valid}"
+                    @click="next">{{ buttonVisibility.next ? "Next" : handleToggledButton}}
+            </button>
+        </div>
+    </div>
+</template>
+
+<script lang="ts">
+import Vue from "vue"
+
+interface Props {
+    buttonVisibility: {}
```

Comment:
Dict from string -> bool?

Reply:
Just suggesting that this field could have a type annotation e.g. `Record<string, boolean>`